recipes/checkm-genome/meta.yaml

```diff
@@ -11,19 +11,19 @@ source:
 
 build:
   number: 0
-  skip: True # [not py27]
+  skip: True # [not py27 or not osx]
 
 requirements:
   build:
-    - python
+    - python <3
```

Comment:
You don't need the pinning here as you already added skipping above.

Reply:
Copy/Paste error solving issue like the 1.0.7 version that uses python3 when used with conda3